code/modules/ninja/admin_ninja_verbs.dm

```diff
@@ -1 +1,59 @@
-// TODO
\ No newline at end of file
+
+/*
+
+Contents:
+- Admin procs that make ninjas
+
+*/
+
+
+//ADMIN CREATE NINJA (From Player)
+/client/proc/cmd_admin_ninjafy(mob/living/carbon/human/H in player_list)
+	set category = null
+	set name = "Make Space Ninja"
+
+	if(!ticker.mode)
+		alert("Wait until the game starts")
+		return
+
+	if(!istype(H))
+		return
+
+	if(alert(src, "You sure?", "Confirm", "Yes", "No") != "Yes")
+		return
+
+	log_admin("[key_name(src)] turned [H.key] into a Space Ninja.")
+	H.mind = create_ninja_mind(H.key)
+	H.mind_initialize()
+	H.equipOutfit(/datum/outfit/ninja)
+	if(istype(H.wear_suit, /obj/item/clothing/suit/space/space_ninja))
+		var/obj/item/clothing/suit/space/space_ninja/NS = H.wear_suit
+		spawn(0)
+			NS.ninitialize(10, H)
+	ticker.mode.update_ninja_icons_added(H)
+
+
+//ADMIN CREATE NINJA (From Ghost)
+/client/proc/send_space_ninja()
+	set category = "Event"
+	set name = "Spawn Space Ninja"
+	set desc = "Spawns a space ninja for when you need a teenager with attitude."
+	set popup_menu = 0
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+	if(!ticker.mode)
+		alert("The game hasn't started yet!")
+		return
+	if(alert("Are you sure you want to send in a space ninja?",,"Yes","No") == "No")
+		return
+
+	var/client/C = input("Pick character to spawn as the Space Ninja", "Key", "") as null|anything in clients
+	if(!C)
+		return
+
+	new /datum/event/ninja
```

Comment:
Don't forget the brackets on this or BYOND gets pissy. Also, `C` doesn't appear to actually be used anywhere.